modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -68,59 +68,74 @@ For more information on systemd settings, see link:https://www.freedesktop.org/s
 +
 // Defaults from https://github.com/openshift/openshift-ansible/pull/3753/files#diff-40b7a7231e77d95ca6009dc9bcc0f470R33-R34
 
-. Convert the `journal.conf` file to base64:
+. Convert the `journal.conf` file to base64 by running the following command:
 +
 [source,terminal]
 ----
-$ export jrnl_cnf=$( cat /journald.conf | base64 -w0 )
+$ export jrnl_cnf=$( cat journald.conf | base64 -w0 )
 ----
 
-. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step.
+. Create a `MachineConfig` object that includes the `jrnl_cnf` variable, which is the encoded contents of the `journald.conf` you created in the previous step. Following is the sample command for worker:
 +
-For example:
+[NOTE]
+====
+You can create a MachineConfig object for both master and worker machine types.
+====
 +
 [source,terminal]
 ----
-$ cat > /tmp/40-worker-custom-journald.yaml <<EOF
+$ cat << EOF > ./99-worker-journald-configuration.yaml <1>
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker
-  name: 50-corp-journald
+    machineconfiguration.openshift.io/role: worker <2>
+  name: 99-worker-journald-configuration <3>
 spec:
   config:
     ignition:
-      version: 3.1.0
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
     storage:
       files:
       - contents:
-          source: data:text/plain;charset=utf-8;base64,${jrnl_cnf}
-        mode: 0644 <1>
-        overwrite: true
-        path: /etc/systemd/journald.conf <2>
+          source: data:text/plain;charset=utf-8;base64,${jrnl_cnf} <4>
+          verification: {}
+        filesystem: root
+        mode: 0644 <5>
```

Comment:
Please also check with QE.  I believe that customers specify the value in decimal and it is converted to octal.

> mode (integer): the file's permission mode. Note that the mode must be properly specified as a decimal value (i.e. 0644 -> 420). If not specified, the permission mode for files defaults to 0644 or the existing file's permissions if overwrite is false, contents.source is unspecified, and a file already exists at the path.

That text is from <https://github.com/coreos/ignition/blob/main/docs/configuration-v3_2.md>.

Reply:
This is already tested and approved by QE for both master and worker as you can see above. I can recheck if that's needed Here's the existing content which has the same description. QE has approved this change.
@anpingli  - Can you confirm this?

Reply:
@mikemckiernan   You can not use 640.  0640 works.   When I set mode: 0666. The perm can be set correctly on the node.

Reply:
I was under the impression that MCO required setting the value in decimal only.  I seems that if a leading zero is specified, then the value is interpreted as octal.

<https://github.com/openshift/machine-config-operator#applying-configuration-changes-to-the-cluster>